src/psyclone/generator.py

```diff
@@ -509,10 +509,7 @@ def main(args):
 
     # Check API, if none is specified, take the setting from the config file
```

Comment:
This comment needs updating but if that's all there is left to do, I shall do it.

Reply:
I've done it.